sdk/turing/version.py

```diff
@@ -1 +1 @@
-VERSION = "0.1.0"
+VERSION = "0.3.0"
```

Comment:
I think I forgot to update this for the release of Turing `0.2.0` but it's now going to updated as usual.

Reply:
How often do we manually update it? And what would be the benefit of doing so? I wonder if this can just be left at `0.0.0` always, since the `make version` command overwrites this file with the correct version during build anyway.

Reply:
> How often do we manually update it?

I think judging from... how the original string was `0.1.0`, and that's our first release... we've probably never updated it at all 😅

>And what would be the benefit of doing so? I wonder if this can just be left at 0.0.0 always, since the make version command overwrites this file with the correct version during build anyway.

That is true. Maybe I'll just roll it back to `0.0.0` like you suggested, and leave a comment to say that's no need to update that field manually.